Added description for implement tap digitizing mode (#447)
opengisch/QField-docs: documentation/how-to/digitize.en.md

!!! note
    This feature is available on Android only.

> Additionally Finger Tap Digitizing Mode in QField, offer to users a swift method to add vertices by tapping on the canvas, balancing speed and functionality. Accessed through the settings panel, this mode caters to scenarios where rapid data input is crucial.

Reviewer: I believe we try to use "you" instead of "users" throughout our documentation.

```suggestion
Additionally, QField has a _finger tap_ digitizing mode where vertices are added by tapping on the canvas. Activated through the settings panel, this mode caters to scenarios where rapid data input is crucial.
```

Author: Completely agree!

> !![](../assets/images/activate_finger_tap_digitizing.png)

Reviewer: Great looking screenshot!
